packages/ckeditor5-paste-from-office/src/filters/removeinvalidtablewidth.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module paste-from-office/filters/removexmlns
```

Comment:
The module name is inconsistent with the file name (CI will fail).

Reply:
Fixed :heavy\_check\_mark: